### DISCUSSION TITLE
Network error on VPN

### DISCUSSION THREAD

**Discussion by bermount** [libre-tube/LibreTube#6700] [General]
### Steps to reproduce

1. Turn on any VPN
2. Home tab shows "Nothing here." with flash "Network error" & When trying to change Instance, it flashes "Could not fetch available instances."
Tried both 'Disable Piped proxy' and 'Local stream extraction' on and off, but didn't show any differences. 
When connected to hotspot from another device using same VPN, it works fine so I don't think VPN or Instance is a problem. 
Assume it's simmilar issue with #775 ?

### Expected behavior

Page to be loaded

### Actual behavior

Network error and doens't load any pages. 

### LibreTube version

0.25.2

### Android version

Android 14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by bermount** [comment]
For anyone facing the same problem, I found out the reason:
I'm using Galaxy with One UI 6 and when connected to VPN, it probably thinks it's not wifi -> when set to only use wifi for a specific app in the settings, it'll show no connection.

**Comment by nutuxedo** [comment]
Yeah I do have the same issue (sorry for the necro bump)

running Proton VPN thru Galaxy A33 5G One UI 7.

I've tweaked tbe settings including disable proxy for the Piped instance and yet to no avail. This is for version
0.28.1
